src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+import java.text.DecimalFormat;
+
+public class RightTriangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+    private String color;
+
+    public RightTriangle() {
+        this.firstLeg = 4;
+        this.secondLeg = 5;
+        this.color = "green";
+    }
```

Comment:
Figure classes shouldn't be bothered with the generation of any values, it's a model, and we should only provide a way to create a figure. FigureSupplier will create these figures with random values via constructors

Pass parameters to the constructor, don't hardcode anything